tools/vrops.py

```diff
@@ -7,7 +7,7 @@
 import os
 
 
-class Resources:
+class Vrops:
```

Comment:
Can we rename the file starting with uppercase `V`?

Reply:
Indeed.